testing/kubernetes/install-kubernetes.sh

```diff
@@ -20,21 +20,21 @@ export K8S_VERSION="v1.5.2"
 export ARCH=amd64
 
 docker run -d \
-    --volume=/sys:/sys:rw \
+    --volume=/:/rootfs:ro \
+    --volume=/sys:/sys:ro \
     --volume=/var/lib/docker/:/var/lib/docker:rw \
-    --volume=/var/lib/kubelet/:/var/lib/kubelet:rw,shared \
+    --volume=/var/lib/kubelet/:/var/lib/kubelet:rw \
     --volume=/var/run:/var/run:rw \
     --net=host \
     --pid=host \
     --privileged \
     --name=kubelet \
     gcr.io/google_containers/hyperkube-${ARCH}:${K8S_VERSION} \
     /hyperkube kubelet \
+        --containerized \
         --hostname-override=127.0.0.1 \
         --api-servers=http://localhost:8080 \
         --config=/etc/kubernetes/manifests \
-        --cluster-dns=10.0.0.10 \
-        --cluster-domain=cluster.local \
         --allow-privileged --v=2
```

Comment:
Saw some problems with the go http client timing out.  Removed the lines (to match what the previous config was) and they went away.